astropy/constants/iau2015.py

```diff
@@ -22,73 +22,73 @@ class IAU2015(Constant):
 GM_sun = IAU2015('GM_sun', 'Nominal solar mass parameter', 1.3271244e20,
-                  'm3 / (s2)', 0.0, "IAU 2015 Resolution B 3", system='si')
+                 'm3 / (s2)', 0.0, "IAU 2015 Resolution B 3", system='si')
 
 # Solar mass (derived from mass parameter and gravitational constant)
-M_sun = IAU2015('M_sun', "Solar mass", GM_sun.value / G.value,
-                 'kg', ((G.uncertainty / G.value) *
-                        (GM_sun.value / G.value)),
-                 "IAU 2015 Resolution B 3 + CODATA 2014", system='si')
+M_sun = IAU2015('M_sun', "Solar mass", GM_sun.value / codata.G.value,
+                'kg', ((codata.G.uncertainty / codata.G.value) *
+                       (GM_sun.value / codata.G.value)),
+                "IAU 2015 Resolution B 3 + CODATA 2014", system='si')
 
 # Solar radius
 R_sun = IAU2015('R_sun', "Nominal solar radius", 6.957e8, 'm', 0.0,
-                 "IAU 2015 Resolution B 3", system='si')
+                "IAU 2015 Resolution B 3", system='si')
 
 
 # OTHER SOLAR SYSTEM QUANTITIES
 
 # Jupiter mass parameter
 GM_jup = IAU2015('GM_jup', 'Nominal Jupiter mass parameter', 1.2668653e17,
-                  'm3 / (s2)', 0.0, "IAU 2015 Resolution B 3", system='si')
+                 'm3 / (s2)', 0.0, "IAU 2015 Resolution B 3", system='si')
 
 # Jupiter mass (derived from mass parameter and gravitational constant)
-M_jup = IAU2015('M_jup', "Jupiter mass", GM_jup.value / G.value,
-                 'kg', ((G.uncertainty / G.value) *
-                        (GM_jup.value / G.value)),
-                 "IAU 2015 Resolution B 3 + CODATA 2014", system='si')
+M_jup = IAU2015('M_jup', "Jupiter mass", GM_jup.value / codata.G.value,
+                'kg', ((codata.G.uncertainty / codata.G.value) *
+                       (GM_jup.value / codata.G.value)),
+                "IAU 2015 Resolution B 3 + CODATA 2014", system='si')
 
 # Jupiter equatorial radius
 R_jup = IAU2015('R_jup', "Nominal Jupiter equatorial radius", 7.1492e7,
-                 'm', 0.0, "IAU 2015 Resolution B 3", system='si')
+                'm', 0.0, "IAU 2015 Resolution B 3", system='si')
 
 # Earth mass parameter
 GM_earth = IAU2015('GM_earth', 'Nominal Earth mass parameter', 3.986004e14,
-                  'm3 / (s2)', 0.0, "IAU 2015 Resolution B 3", system='si')
+                   'm3 / (s2)', 0.0, "IAU 2015 Resolution B 3", system='si')
 
 # Earth mass (derived from mass parameter and gravitational constant)
 M_earth = IAU2015('M_earth', "Earth mass",
-                   GM_earth.value / G.value,
-                 'kg', ((G.uncertainty / G.value) *
-                        (GM_earth.value / G.value)),
-                 "IAU 2015 Resolution B 3 + CODATA 2014", system='si')
+                  GM_earth.value / codata.G.value,
+                  'kg', ((codata.G.uncertainty / codata.G.value) *
+                         (GM_earth.value / codata.G.value)),
+                  "IAU 2015 Resolution B 3 + CODATA 2014", system='si')
```

Comment:
The docstring should read `+ CODATA 2018` for the defaults. I agree with @mhvk that the reference should be taken from `G`. Does this work?

```suggestion
                  "IAU 2015 Resolution B 3 + {}".format(codata.G.reference), system='si')
```

Reply:
Oh, ops. I came to the same conclusion before I saw this suggestion. But yes, that looks great. Thanks!